### PR TITLE
Added postgresql rivery and zapier connect how-to

### PR DESCRIPTION
**Pull request by runwuf**
Added postgresql rivery and zapier connect how-to

### CODE REVIEW

**Review by ftisiot — CHANGES_REQUESTED** [review]
Hi @runwuf, can you please check the warnings? You should probably add `Rivery` to `.github/vale/dics/aiven.dic` and also in `.github/vale/styles/capitalization_headings.yml` if you plan to use it with capital `R` in titles

**Comment by runwuf** [comment]
> Hi @runwuf, can you please check the warnings? You should probably add `Rivery` to `.github/vale/dics/aiven.dic` and also in `.github/vale/styles/capitalization_headings.yml` if you plan to use it with capital `R` in titles

Thanks @ftisiot, fixed these warnings.

File: docs/products/postgresql/howto/connect-zapier.rst [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+'''''''''''''''''''''
+
+1. In skyvia workspace -> **Connections** -> **Connector** -> **PostgreSQL**.
+2. In the **General** Tab give the connection a Name (e.g. ``MyDatabase``)
+3. In the **Connection** tab set:
+
+   * **Host** to ``HOSTNAME``
+   * **Port**: to ``PORT``
+   * **Database** to ``DATABASE``
+   * **Schema** to ``public``
```
Comment (ftisiot) [inline docs/products/postgresql/howto/connect-zapier.rst:32]: can we have a `SCHEMA` placeholder as well to make this more generic?

**Review by ftisiot — CHANGES_REQUESTED** [review]
Minor change needed, very good job @runwuf !

File: docs/products/postgresql/howto/connect-zapier.rst (hunk 1 above)
Comment (runwuf) [reply docs/products/postgresql/howto/connect-zapier.rst]: done.

**Review by ftisiot — APPROVED** [review]
LGTM, thanks!

**Pull request merged**